wikitemplate-cr-upgrade.md

```diff
@@ -69,6 +69,7 @@ _Each start should take less than 7 seconds_
   - [ ] opened tabs can be reloaded
   - [ ] bookmarks on the bookmark toolbar can be opened
   - [ ] bookmarks in the bookmark folder toolbar can be opened
+  - [ ] ensure upgrade sets fingerprint to `Block all fingerprint` when global fingerprint is enabled
```

Comment:
Do we only need to check this on update to 0.20.x?

Reply:
Can you specifically mention `0.19.x` -> `0.20.x`. After a few `0.20.x` hotfixes, we can eventually remove this case as most people would have already migrated from `0.19.x` -> `0.20.x`.

Reply:
Sure will add upgrade from 01.9.x to 0.20.x